src/doc/functions_to_method_doc.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python
+
+copyright = '/*****************************************************************************\n'
```

Comment:
Lets put this copyright text into its own text file and slurp it into a python variable whenever we need it or

Reply:
I thought there was some resolution to this that meant we didn't have to have the copyright at the top of every single file. Is that accurate? If so, then I can just remove this variable.

Reply:
Yeah, that is #12 . But, we need to have that copyright verbiage in a top-level COPYRIGHT file *and* you would need to include the `See top level COPYRIGHT file for copyright information` comment in this file as a replacement for all this copyright text.

If we do that in this PR, then our source code would have a sort of hybrid of the two approaches; the new approach here and the old stuff everywhere else.

Not sure what is best. Given this discussion though, am fine with you leaving it as is for now.